### DISCUSSION TITLE
user local storage info

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#4904] [Q&A]
i have the typical user table but with a balance, i listen to the balance if it is update from the admin ui like this
```
useEffect(() => {
    setBalance(initialBalance);
    function subscribeCallback(e: any) {
      setBalance(e.record.balance);
      console.log(e.record);
    }

    pb.collection("users").subscribe(userId, subscribeCallback);

    return () => {
      pb.collection("users").unsubscribe("*");
    };
  }, [userId, initialBalance]);
```
  this updates it but the local storage data isnt updated and i think it is better to fetch the user balance from the local storage due to it being faster, do i update it using authRefresh? or is it better to fetch balance each time

**Comment by ganigeorgiev** [comment]
I'm not entirely sure that I understand your use case but you can manually update the auth store by calling `save` in your `subscribeCallback`:

```js
pb.authStore.save(pb.authStore.token, e.record)
```
